cosmwasm/contracts/global-accountant/src/contract.rs

```diff
@@ -334,9 +334,15 @@ fn handle_vaa(
     let mut evt = if body.emitter_chain == Chain::Solana
         && body.emitter_address == wormhole::GOVERNANCE_EMITTER
     {
```

Comment:
If we want to still handle the errors from parsing, we could do something like:

```rust
    if body.payload.len() < 32 {
        bail!("governance module missing");
    }
    let module = &body.payload[..32];

    if module == TOKENBRIDGE_MODULE {
        let govpacket = serde_wormhole::from_slice(body.payload)
            .context("failed to parse tokenbridge governance packet")?;
        handle_token_governance_vaa(deps.branch(), body.with_payload(govpacket))
    } else if module == ACCOUNTANT_MODULE {
        let govpacket = serde_wormhole::from_slice(body.payload)
            .context("failed to parse accountant governance packet")?;
        handle_accountant_governance_vaa(deps.branch(), body.with_payload(govpacket))
    } else {
        bail!("unknown governance module")
    }
```

We could export the constants for the tokenbridge and accountant governance modules from the rust SDK.

Reply:
extra safe, will add it now.